README.md

```diff
@@ -54,6 +54,11 @@ Add specifications for additional contents managers in your user settings (in th
       "name": "s3 test bucket",
       "url": "s3://test"
     },
+    {
+      "name": "s3 test key",
+      "url": "s3://test-2/prefix/",
+	  "defaultWritable": false
```

Comment:
```suggestion
      "name": "s3 test key",
      "url": "s3://test-2/prefix/",
      "defaultWritable": false
```

use the same indentation level for `"defaultWritable"` here, to match with the other properties. Also, use spaces not tabs